Don't compile portAudio output plugin on Linux.
And properly detect include paths using pkg-config.
--- a/LastFM.pro
+++ b/LastFM.pro
@@ -41,8 +41,7 @@
     SUBDIRS -= src/Twiddly \
                src/Bootstrapper/ITunesDevice
 
-    SUBDIRS += src/output/alsa-playback \
-               src/output/portAudio
+    SUBDIRS += src/output/alsa-playback
 }
 
 
--- a/src/mediadevices/ipod/ipod.pro
+++ b/src/mediadevices/ipod/ipod.pro
@@ -11,7 +11,7 @@
 linux* {
     LIBS += -lgpod
 
-    INCLUDEPATH += /usr/include/gpod-1.0 /usr/include/glib-2.0 /usr/lib/glib-2.0/include
+    INCLUDEPATH += $$system(pkg-config --cflags-only-I libgpod-1.0 | sed -e's/-I//g')
 }
 
 mac {
